modules/rixengineBidAdapter.js

```diff
@@ -4,14 +4,16 @@ import { registerBidder } from '../src/adapters/bidderFactory.js';
 
 const BIDDER_CODE = 'rixengine';
 
-let ENDPOINT = null;
-let SID = null;
-let TOKEN = null;
-
+// We will also allow an alias named 'algorix'
+const ALGORIX_REGIONS = ['apac', 'use', 'euc'];
```

Comment:
I didn't implement this, but you could detect the region like this https://github.com/prebid/Prebid.js/blob/698ede8a1cca503d978095439d77df78931e24e6/libraries/smartyadsUtils/getAdUrlByRegion.js#L14